jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitProjectBuilderCommon.kt

```diff
@@ -28,20 +28,36 @@ abstract class SamProjectTemplate {
     fun getIcon() = AwsIcons.Resources.SERVERLESS_APP
 
     fun build(runtime: Runtime, outputDir: VirtualFile) {
-        doBuild(runtime, outputDir)
-        TelemetryService.getInstance().record("SamProjectInit") {
-            datum(getName()) {
-                metadata("runtime", runtime.name)
-                metadata("samVersion", SamCommon.getVersionString())
+        try {
+            doBuild(runtime, outputDir)
+        } finally {
+            TelemetryService.getInstance().record("SAM") {
+                datum("Init") {
+                    metadata("name", getName())
+                    metadata("runtime", runtime.name)
+                    metadata("samVersion", SamCommon.getVersionString())
```

Comment:
Add a metadata flag to track exceptions

https://github.com/aws/aws-toolkit-jetbrains/blob/83216e60998f6adf02a2fb7052bb4f602bcae214/jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamInvokeRunner.kt#L124